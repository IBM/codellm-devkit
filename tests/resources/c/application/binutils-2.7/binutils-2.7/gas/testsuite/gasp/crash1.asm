

	.MACRO 	foo a b c=a
	\a \b \c \d
	.ENDM

	foo 1 2
	foo 1 2 3 4
	foo 1
	foo 


	.END
