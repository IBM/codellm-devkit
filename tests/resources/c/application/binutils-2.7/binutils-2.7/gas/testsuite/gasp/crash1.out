!

!

!	.MACRO 	foo a b c=a
!	\a \b \c \d
!	.ENDM
!

!	foo 1 2
!	1 2 a \d
	1 2 a \d
!	foo 1 2 3 4
!	foo 1
!	1  a \d
	1  a \d
!	foo 
!	  a \d
	a \d
!

!

!	.END
